modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -405,6 +405,11 @@
 		<td>//div[contains(@class,'dnd-th') and contains(text(),'Type')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>FOLDER_LABEL</td>
+		<td>//span[@class='lfr__object-web-view-object-definitions-title mb-0' and contains(text(),'${folderLabel}')]</td>
```

Comment:
This path looks fragile, why don't we assert part of the class instead of the exact class here? Can we just use `view-object-definitions`?

Reply:
Yep, I'll do the updates